commcare_connect/opportunity/tests/test_task_completion.py

```
@@ -0,0 +1,96 @@
+
+CALLBACK_URL = "/api/task_completed/"
+
+
+def _ocs_task(**kwargs):
+    return AssignedTaskFactory(task_type__mode=TaskTypeModeChoices.OCS, **kwargs)
+
+
+def _authed_client(user):
+    client = APIClient()
+    token, _ = user.oauth2_provider_accesstoken.get_or_create(
+        token="task-callback-token",
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "token"

(S106)